### PR TITLE
[MTD] Clarification Dashboard Update

### PR DESCRIPTION
**Pull request by ethan-carlson**
# Description

This PR includes the updates to the dashboard to support graceful clarification.  It also includes some quality-of-life improvements to the chat interface source code for legibility.

See the screenshots below for the changes.

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Proposes a change (non-breaking change that isn't necessarily a bug)
- [ ] Refactor
- [X] New feature (non-breaking change that adds a new functionality)
- [ ] Breaking change (fix or feature that would break some existing functionality downstream)
- [ ] This is a unit test
- [ ] Documentation only change
- [ ] Datasets Release
- [ ] Models Release

## Type of requested review

- [ ] I want a thorough review of the implementation.
- [X] I want a high level review. 
- [ ] I want a deep design review.

### Starting State
Old
start_main
New
start_new

### Status Updates
Old
status_main
New
status_new

### Go Back Buttons
Old
go_back_main
New
go_back_new

### Reset / Stop Button Placement
Old
reset_main
New
reset_new

### Response Options
Old
options_main
New
options_new

# Testing

I've tested this change thoroughly, both dashboard locally and in mock turk jobs.

To see the new features, open the dashboard and issue commands to the agent that trigger clarification (eg. destroy something that doesn't exist).

# Checklist:

- [X] I have performed manual end-to-end testing of the feature in my environment.
- [X] I have added Docstrings and comments to the code.
- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [X] New and existing unit tests pass locally with my changes.
- [X] I have added relevant collaborators to review the PR before merge.
- [ ] [Polymetis only] I ran on hardware (1) all scripts in `tests/scripts`, (2) asv benchmarks.

### CODE REVIEW

**Review by kavyasrinet — COMMENTED** [review]
I reviewed the code closely and the implementation looks good but `InteractApp.js` is giant enough that I think it deserves a README / design/architecture doc. The code as is, is well documented but anyone new contributing to this file will definitely need some background. Could you add that doc to this PR ?

**Comment by ethan-carlson** [comment]
> I reviewed the code closely and the implementation looks good but `InteractApp.js` is giant enough that I think it deserves a README / design/architecture doc.

@kavyasrinet added!  Let me know if you think it's OK or if you have any other edits.

**Comment by kavyasrinet** [comment]
> > I reviewed the code closely and the implementation looks good but `InteractApp.js` is giant enough that I think it deserves a README / design/architecture doc.
> 
> @kavyasrinet added! Let me know if you think it's OK or if you have any other edits.

Thanks for adding this, this looks good. In addition, I think it'll be useful to add something like : https://github.com/facebookresearch/fairo/pull/934/files at the folder level as well. Some way to capture the flowchart of the workflow

File: droidlet/dashboard/web/src/components/Interact/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+## Chat Architecture Flowchart ##
+<pre>
+`handleSubmit`                            User sends a new chat (command status: 'sent')
```
Comment (kavyasrinet) [inline droidlet/dashboard/web/src/components/Interact/README.md:3]: nit that "`" doesn't render properly inside "<"pre">" tags.

**Review by kavyasrinet — APPROVED** [review]
Thanks for updating the README, this looks much better.

**Pull request merged**